net.pcsx2.PCSX2.yml

```diff
@@ -74,10 +76,22 @@ modules:
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
-      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-      - -DDISABLE_BUILD_DATE=ON
-      - -DDISABLE_ADVANCE_SIMD=ON
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DQT_BUILD=ON
+      - -DCUBEB_API=ON
+      - -DX11_API=ON
+      - -DWAYLAND_API=ON
+      - -DENABLE_SETCAP=OFF
+      - -DUSE_SYSTEM_SDL2=ON
+      - -DUSE_SYSTEM_ZSTD=OFF
+      - -DDISABLE_ADVANCE_SIMD=TRUE
+      ##############################################################
```

Comment:
also stuff below this line are defined in a different way in the pcsx2 build scripts, so I put this separator for ease for readability.
https://github.com/PCSX2/pcsx2/blob/1067ed309a035d984c665f6b226f30d0c6454e46/.github/workflows/scripts/linux/generate-cmake-qt.sh#L5

Reply:
also in their build script thankfully there are no ${ADDITIONAL_CMAKE_ARGS}
as shown there:
https://github.com/PCSX2/pcsx2/blob/1067ed309a035d984c665f6b226f30d0c6454e46/.github/workflows/linux_build_matrix.yml#L18
or there:
https://github.com/PCSX2/pcsx2/blob/1067ed309a035d984c665f6b226f30d0c6454e46/.github/workflows/release_pipeline.yml#L18
"cmakeflags:" is where the ${ADDITIONAL_CMAKE_ARGS} variable is set from.